common/app/views/fragments/exploreSeries.scala.html

```diff
@@ -45,7 +45,7 @@
                 @if(page.item.elements.hasMainVideo) {
                   @fragments.immersiveVideo(page)
                 }
-                @if(page.item.elements.hasMainEmbed) {
+                @if(hasMainMedia && !page.item.elements.hasMainVideo) {
```

Comment:
`page.item.elements.hasMainVideo` can be replaced by `hasVideo` (defined on lines 13-14)